easybuild/easyconfigs/s/SIMPLE/SIMPLE-2.5-foss-2018a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'SIMPLE'
+version = '2.5'
+
+homepage = 'http://simplecryoem.com/'
+description = """ Single-particle IMage Processing Linux Engine is a program package
+ for cryo-EM image processing, focusing on ab initio 3D reconstruction of low-symmetry single-particles. """
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['http://simplecryoem.com/downloads']
+sources = ["%(namelower)s%(version)s.tgz"]
+checksums = ['e3a25193151bf54eabebd5d775161ef6ae9996fa9c8d523649d26957cbc181b1']
+
+builddependencies = [('CMake', '3.10.1')]
+
+separate_build_dir = True
+
+modextravars = {
+    'SIMPLE_QSYS': 'local',
+    'SIMPLE_PATH': '%(installdir)s',
+}
+
+modextrapaths = {'PATH': 'scripts'}
+
+sanity_check_paths = {
+    'files': ['bin/simple_distr_exec', 'bin/simple_exec',
+              'lib/libSIMPLE2.5.a', 'lib/simple/simple_varlist.txt'],
```

Comment:
@smoors Please use the `%(version)s` template rather than hardcoding `2.5`: `'lib/libSIMPLE%(version)s.a'`